src/client/pps/pps.proto

```diff
@@ -370,4 +373,7 @@ service API {
   // DeleteAll deletes everything
   rpc DeleteAll(google.protobuf.Empty) returns (google.protobuf.Empty) {}
   rpc GetLogs(GetLogsRequest) returns (stream LogMessage) {}
+
+  // Garbage collection
+  rpc GC(GCRequest) returns (GCResponse) {}
```

Comment:
I don't have a particular strong argument for why this is better, but I like the consistency/extensibility of it.  For instance, when I was iterating on this PR, I had to update certain message types many times (e.g. `ListTagsRequest/Response`).  It felt good that I only had to add/remove fields within the messages, as opposed to changing the message types themselves from/to `Empty`.

In this case in particular, it's very possible that we will be adding options to `GCRequest` soon, and I had plans for making `GCResponse` into a stream that reports progress (since GC can take a long time).  Now, why can't we just use `Empty` for now and change them to specific types later?  Again I don't have a good answer.  I guess this just comes down to personal preference.

I also read somewhere that having dedicated request/response types for each API is a good practice, but that's of course debatable but at least I'm not the only person who likes it that way.